syntax = "proto3";

message Building {
  string name = 1;
  string buildingNumber = 2;
  Street street = 3;

  message Street {
    string name = 1;
    City city = 2;
  }

  message City {
    string name = 1;
    string zipCode = 2;
    string countryName = 3;
  }
}